sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/DynamicDestinationsHelpers.java

```diff
@@ -221,6 +221,48 @@ public String toString() {
     }
   }
 
+  static class ConstantClusteringDestinations<T> extends ConstantTimePartitioningDestinations<T> {
```

Comment:
Possibly cleaner rename and update existing class instead of sub-classing here.

Reply:
done